### PR TITLE
COMMERCE-11392

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-11392](https://issues.liferay.com/browse/COMMERCE-11392)

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 1]
```diff
@@ -2849,6 +2849,62 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11392. Can see timestamps from payment transaction history"
+	@priority = 3
+	test CanViewOrderTransactionHistoryTimestamps {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2854]: I would change the name into CanViewTimestampsInOrderTransactionHistory

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 2]
```diff
@@ -2849,6 +2849,62 @@ definition {
...
+		property portal.acceptance = "true";
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2855]: I see you set portal.acceptance = "true". Are we gonna add the test to the relevant?

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 3]
```diff
@@ -2849,6 +2849,62 @@ definition {
...
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Commerce Account",
+				accountEntryType = "Business");
+		}
+
+		task ("And a user checks out an order ") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2865]: I would say "And the user ... "

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 4]
```diff
@@ -2849,6 +2849,62 @@ definition {
...
+				userEmailAddress = "test@liferay.com");
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = 1);
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+
+		CommerceCheckout.initializersCheckout(
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2880]: Is this outside of the task? i think it should go inside the curly braces of the task

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 5]
```diff
@@ -2849,6 +2849,62 @@ definition {
...
+			newAddressCity = "Test City",
+			newAddressCountry = "United States",
+			newAddressName = "Address Name",
+			newAddressZipCode = "Test Zip Code",
+			productName = "U-Joint",
+			productQuantity = 1,
+			productUnitPrice = 24,
+			shippingMethod = "Standard");
+
+		task ("When a user opens the Payments tab") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2891]: I would say "When the user ... "

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 6]
```diff
@@ -2849,6 +2849,62 @@ definition {
...
+			CommerceOrders.openOrdersAdmin();
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "Pending");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Payments");
+		}
+
+		task ("Then a user can view the Transaction History") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2899]: "Then the user ... "

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**